db/DB.js

```diff
@@ -8,22 +10,29 @@ const knexConfig = require("./knexfile");
 const HIBP = require("../hibp");
 const getSha1 = require("../sha1-utils");
 
-const knex = Knex(knexConfig);
+const knex = Knex(knexConfig[AppConstants.NODE_ENV]);
```

Comment:
Do we need to add more safety around this? Like, if `process.env.NODE_ENV` is set to "production" (instead of "prod"), the `knexConfig[AppConstants.NODE_ENV]` would presumably be `undefined`, but I don't know what that would do if we pass `undefined` to the `Knex()` constructor.
Wasn't sure if storing `knexConfig[AppConstants.NODE_ENV]` in a temporary variable and checking for truthiness (else failing hard) would be worth the extra hassle or not.

Reply:
It results in a knex connection error which kills the server or tests process pretty early. Which seems like as good a time as any for an error to throw.